helpers/mime.js

```diff
@@ -34,14 +33,9 @@ const KNOWN_FILE_EXTS_IN_TEXT_FORMAT = [
  * @method getMimeFromFileName
  * @param  {String}  fileExtension  File extension (e.g. css, txt, html)
  * @param  {String}  fileName       File name      (e.g. dockerfile, main)
- * @param  {Boolean} raw            raw data type  (e.g. application/javascript instead of text/javascript)
  * @return {String}  MIME Type      eg. text/html, text/plain
  */
-function getMimeFromFileName(fileExtension, fileName = '', raw = false) {
-    if (raw === true) {
-        return mime.lookup(fileExtension) || '';
-    }
-
+function getMimeFromFileName(fileExtension, fileName = '') {
```

Comment:
This does not look correct, we are relying on file extension. This violates security best practice of validating file extensions and names

Reply:
I removed the first condition check, it is still in line 53
![image](https://github.com/screwdriver-cd/store/assets/15989893/71756a91-07d2-4eb2-a2c0-c28be0f2745d)